paf-mvp-core-js/src/model/generated-model.ts

```diff
@@ -359,6 +365,7 @@ export interface OpenRtbBidRequest {
         };
       };
     };
+    [k: string]: unknown | undefined;
```

Comment:
It can be removed safely, but as we discussed offline,
1. this file is generated by script so removing some classes would require either doing it manually at each run, or editing the script to ignore these classes
2. with typescript modules, unused classes are not bundled so there is no waste of memory

As this is unrelated to this change, I think this should be done in a separate PR and shouldn't block this one.